db/migrate/20100310080727_add_family_to_os.rb

```diff
@@ -1,29 +1,6 @@
 class AddFamilyToOs < ActiveRecord::Migration[4.2]
-  class Operatingsystem < ApplicationRecord; end
-
   def up
     add_column :operatingsystems, :family_id, :integer
-
-    Operatingsystem.reset_column_information
-
-    Operatingsystem.all.each do |os|
-      case os.name
-      when /RedHat|Centos|Fedora/i
-        os.family_id = Operatingsystem::FAMILIES.index :RedHat
```

Comment:
Fun fact: I can't see how this worked. AFAIK Families never had an entry which accepted symbols and it always was `Redhat`, not `RedHat`.